client/my-sites/earn/memberships/index.jsx

```diff
@@ -595,11 +597,11 @@ class MembershipsSection extends Component {
 						) }
 					</div>
 					<div>
-						<h3>{ translate( 'No membership fees' ) }</h3>
-						<p>{ preventWidows( translate( 'No monthly or annual fees charged.' ) ) }</p>
+						<h3>{ translate( 'Simple fees structure' ) }</h3>
```

Comment:
TODO: wrap in the check if translations are available, to be able to ship English copy early.